index.html

```diff
@@ -230,6 +230,84 @@ <h1>Skills</h1>
           </ul>
         </div>
       </section>
+      <section class="contact">
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] You tried implementing the form submission 👍🏽 . Your form is currently not submitted because your form spree URL is lacking an `id` that is unique to you. Checkup your unique id after registering to formspree

Kindly note that your link should follow this format `https://formspree.io/f/maykokbe`

Reply:
So it turns out that A unique ID will be provided only for paid users. So I am just going to remove the submission.

Reply:
Thanks for the suggestion. Done it :-)